.vsts-ci/templates/releaseBuildPhase.yml

```diff
@@ -12,32 +12,89 @@ jobs:
     vmImage: ${{ parameters.vmImage }}
 
   steps:
+
+  - powershell: |
+        sudo chown root:root /
+    displayName: 'Make sure root owns root'
+    condition: succeeded()
+
   - powershell: 'dir env:'
     displayName: 'Capture Environment'
 
+  - powershell: |
+        $channel = (get-content ./snapcraftConfig.json | ConvertFrom-Json).channel
+        Write-Verbose -Verbose -Message "using Channel $channel"
+        sudo snap install snapcraft --classic "--channel=$channel"
+    displayName: Install snapcraft
+    condition: succeeded()
+
   - powershell: |
       $releaseTagName = switch('$(channel)') {
         'stable' {
+          cd stable
           'StableReleaseTag'
         }
+        'lts' {
+          cd lts
+          'LtsReleaseTag'
+        }
         'preview' {
+          cd preview
           'PreviewReleaseTag'
         }
         default {
+          cd preview
           'PreviewReleaseTag'
         }
       }
       Write-Host "releaseTagName: $releaseTagName"
-      $releaseTag = Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json' | Select-Object -ExpandProperty $releaseTagName
+      $releaseTag = Invoke-RestMethod -Uri 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json' | Select-Object -ExpandProperty $releaseTagName -First 1
       Write-Host "$releaseTag"
       Write-Host "##vso[task.setvariable variable=releaseTag;]$releaseTag"
     displayName: Get Latest $(channel) Release Tag
     condition: and(succeeded(), not(startsWith(variables['releaseTag'], 'v')))
 
   - powershell: |
-      ./tools/releaseBuild/vstsbuild.ps1 -ReleaseTag $(releaseTag) -Name powershell-snap-$(channel)
+      switch('$(channel)') {
+        'stable' {
+          cd stable
+        }
+        'lts' {
+          cd lts
+        }
+        'preview' {
+          cd preview
+        }
+        default {
+          cd preview
```

Comment:
would it be better to error if there is a new channel?

Reply:
```suggestion
          throw "Unknown channel: '$(channel)'"
```